src/index.js

```diff
@@ -66,28 +72,29 @@ class showtimes {
       }
 
       // Use the hidden API of getTheaters to pass in the next page and current theaters.
-      api.getTheaters(cb, ++api.page, api.theaters)
+      api.getTheaters(cb, null, ++api.page, api.theaters)
```

Comment:
`query` should get passed in here, instead of null, else your first page will contain filtered results, and the second unfiltered.